msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages_eu.properties

```diff
@@ -891,3 +891,19 @@ perm-msg.permissions.allowToField.users=Pertsona guztiei bidali
 perm-msg.permissions.allowToField.myGroups=Norberaren taldeei bidali
 perm-msg.permissions.allowToField.myGroupMembers=Norberaren taldeen pertsonei bidali
 perm-msg.permissions.allowToField.myGroupRoles=Norberaren taldeen rolei bidali
+
+#S2U-27
+pvt_publish_to_faq=Publish to FAQ
```

Comment:
Hey @pablosanroman, we have a small number of new keys, would you mind translating them? Not in a hurry.

Reply:
Hi @mpellicer!here it is!: 

pvt_title=Titulua
pvt_question=Galdera
pvt_question_title_prefix=Q:
pvt_answer=Erantzuna
pvt_answer_title_prefix=A:
pvt_answer_info=Erantzunaren eremua hutsik utz dezakezu publikatu nahi ez baduzu oraindik. 
pvt_missing_title=Ezinbestekoa da izenburu bat sartzea "Ohiko galderak" atalean publikatzeko. 
pvt_missing_question=Ezinbestekoa da galdera bat sartzea "Ohiko galderak" atalean publikatzeko. 

cdfm_faq_forum_title=FAQ foroa
cdfm_faq_forum_description=Foroa Ohiko galderentzat
cdfm_faq_topic_title=FAQ Gaia
cdfm_faq_topic_description=Gaia Ohiko galderentzat